src/components/Selfie/assets/selfie-picto.svg

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Has this svg been optiomised?

Reply:
no this svg has not been optimised. What does optimising mean in an svg?